readme.md

```diff
@@ -105,6 +111,19 @@ String or array of strings to match.
 
 Type: `object`
 
+##### allPatterns
```

Comment:
I think it should be added last. It's the least important one.

Reply:
I prefer lexicographic orderring for easier navigation, but ok. I'll do the same in index.d.ts as well.